docs/01_nodeos/03_plugins/mongo_db_plugin/index.md

```diff
@@ -1,153 +0,0 @@
-[[warning | Deprecation Notice]]
```

Comment:
@lparisc does removing this file cause any problems?

Reply:
It's fine as long as all references to the file are also removed, which seems to be the case.